### DISCUSSION TITLE
Saving game states with pickle

### DISCUSSION THREAD

**Discussion by slysnivy** [shoc71/dont-stop-now#56] [Ideas]
I think a good idea is to implement saving the game, just so the player can decide to keep their statistics (total deaths and total jumps) and level progress. I found this useful link (https://stackoverflow.com/questions/6420311/how-to-make-save-load-game-functions-in-pygame) for reference and we can build off of it in the future.

**Comment by shoc71** [comment]
that's a good idea. Let's work on that we have confidently published our first copy.
